test/echo_server.erl

```diff
@@ -59,7 +59,6 @@ handle_call(Call, _From, _State) ->
 handle_continue(Continue, _State) ->
     Continue.
 
--spec format_status(normal | terminate, [[{_, _}] | State, ...]) ->
-                       {formatted_state, State}.
-format_status(_, [_PDict, State]) ->
-    {formatted_state, State}.
+-spec format_status(gen_server:format_status()) -> gen_server:format_status().
+format_status(State) ->
+    State.
```

Comment:
This is basically noop. I didn't want to change the state (as we otherwise get warnings about not returning a map), so by losing `{formatted_state` this function exists only to make its existence explicit.